packages/messaging/src/controllers/window-controller.ts

```diff
@@ -102,7 +223,6 @@ export class WindowController implements FirebaseMessaging, FirebaseService {
     }
   }
 
-  // TODO: Deprecate this and make VAPID key a parameter in getToken.
   usePublicVapidKey(vapidKey: string): void {
```

Comment:
Can you please annotate the deprecated methods with `@deprecated` in the source code as well?